source/documentation/deploying_services/route_services.md

```diff
@@ -0,0 +1,108 @@
+## Route services
+
+Tenants may wish to apply some processing to requests before they reach an application. Common examples of use cases are authentication, rate limiting, and caching services.
+
+Cloud Foundry allows the tenants to bind [application routes](/#names-routes-and-domains) to [Route Services](https://docs.cloudfoundry.org/services/route-services.html) [external-link]. A Route Service acts as a full proxy. Once it is bound to a route, the platform routing layer will send every request for that route to the Route Service endpoint. The Route Service can then process the incoming request, proxy it back to the original application, and finally process the response request before returning it to the original client.
```

Comment:
The "[external link]" thing is a pattern that's in use across the docs at the moment for all links that go to the upstream docs. I think we should be consistent about this.